### PR TITLE
WEBUI-397: share saved search popup issue resolved

### PR DESCRIPTION
**Pull request by SnehalNaik-DEV**

### CODE REVIEW

File: elements/search/nuxeo-search-form.js [hunk 1]
```diff
@@ -403,14 +431,24 @@ Polymer({
...
 
-    <nuxeo-dialog id="shareDialog" with-backdrop reparent opened="{{permissionsVisible}}">
-      <h2>[[i18n('searchForm.shared.heading')]]</h2>
-      <nuxeo-document-permissions
-        doc-id="[[selectedSearch.id]]"
-        visible="[[permissionsVisible]]"
-      ></nuxeo-document-permissions>
-      <div class="buttons">
-        <paper-button dialog-dismiss class="secondary">[[i18n('command.close')]]</paper-button>
+    <nuxeo-dialog id="shareDialog" with-backdrop reparent opened="{{permissionsVisible}}" class="sharedialog">
```
Comment (richardsd) [inline elements/search/nuxeo-search-form.js]: @SnehalNaik-DEV although this solution is kind of working, it seems a bit over-complicated. 
We could take this approach and use all this CSS ⬆️ to add scrolling to the panel, but I think we could use a similar solution to what we use in other places (see https://github.com/nuxeo/nuxeo-web-ui/blob/maintenance-3.0.x/addons/nuxeo-template-rendering/elements/nuxeo-delete-template-button.js#L49-L53). Using the `paper-dialog-scrollable` you can wrap up the part you want to be scrollable, without having to mess around with the CSS like this. What do you think?

A good practice is to always look for similar use cases in the application 😉 
paper-dialog-scrollable

**Review by Gabez0r — COMMENTED** [review]
Well done. But now we have two commits, with two different solutions, but we only need one:
image
Please squash the commits.

**Review by rakeshkumar1019 — COMMENTED** [review]
looks good to me.

**Review by Gabez0r — APPROVED** [review]
Good job!

**Review by rakeshkumar1019 — COMMENTED** [review]
looks good

**Review by rakeshkumar1019 — APPROVED** [review]

**Pull request merged**